### PR TITLE
Add integration test for new remove auth command

### PR DESCRIPTION
**Pull request by scotchneat**
This adds integration tests for the new command added in #981

Edit: To run just this new test (locally), I used: 

`go test -v -mod=vendor ./integration/init_test.go ./integration/auth_remove_context_test.go`

Results:
```
=== RUN   TestRun
Suite: doctl
Total: 2 | Focused: 0 | Pending: 0
=== RUN   TestRun/doctl
=== RUN   TestRun/doctl/auth/remove/a_context_is_not_provided/should_error
=== PAUSE TestRun/doctl/auth/remove/a_context_is_not_provided/should_error
=== RUN   TestRun/doctl/auth/remove/a_valid_context_is_provided/allows_you_to_remove_that_context
=== PAUSE TestRun/doctl/auth/remove/a_valid_context_is_provided/allows_you_to_remove_that_context
=== CONT  TestRun/doctl/auth/remove/a_context_is_not_provided/should_error
=== CONT  TestRun/doctl/auth/remove/a_valid_context_is_provided/allows_you_to_remove_that_context

Passed: 2 | Failed: 0 | Skipped: 0

--- PASS: TestRun (1.18s)
    --- PASS: TestRun/doctl (0.00s)
        --- PASS: TestRun/doctl/auth/remove/a_context_is_not_provided/should_error (0.58s)
        --- PASS: TestRun/doctl/auth/remove/a_valid_context_is_provided/allows_you_to_remove_that_context (1.18s)
PASS
ok  	command-line-arguments	4.457s
```

### CODE REVIEW

File: integration/auth_remove_context_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+				"--config", testConfig,
+				"--context",
+				removeContext,
+			)
+			_, err := cmd.CombinedOutput()
+			expect.NoError(err)
+
+			fileBytes, err := ioutil.ReadFile(testConfig)
+			expect.NoError(err)
+			expect.NotContains(string(fileBytes), "context: second")
```
Comment (andrewsomething) [inline integration/auth_remove_context_test.go:71]:
```suggestion
			expect.NotContains(string(fileBytes), "second-token")
```

Looks like the config will never contain `context: second` as written:

```
access-token: first-token
auth-contexts:
  second: second-token
context: default
```

**Review by andrewsomething — CHANGES_REQUESTED** [review]
Thanks for writing these tests!

One comment in-line.

File: integration/auth_remove_context_test.go (hunk 1 above)
Comment (scotchneat) [reply integration/auth_remove_context_test.go:71]: Ah, good catch. I copied that from another test but missed that update.

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM!

**Pull request merged**